functions/go/Makefile

```diff
@@ -117,3 +132,7 @@ func-build: func-verify
 func-push:
 	@echo Pushing image $(CURRENT_FUNCTION)...
 	../../scripts/go-function-release.sh push curated
+
+func-generate:
```

Comment:
this is not `generating the function` just the docs right?

Reply:
Renamed it to docs-generate